api/v2/openapi.yaml

```diff
@@ -505,6 +542,29 @@ definitions:
       - state
       - silencedBy
       - inhibitedBy
+  extendedAlertStatus:
+    type: object
+    properties:
```

Comment:
I've also had to duplicate this otherwise the `state` enum does not get inherited and the Elm code fails.